examples/hello/src/logging.rs

```diff
@@ -0,0 +1,114 @@
+use std::backtrace::Backtrace;
+use std::env;
+use std::panic::PanicHookInfo;
+
+use tracing::Event;
```

Comment:
May we use log + logforth (+fastrace)?

Reply:
Seems we are using tracing everywhere inside suraft. Perhaps revisit this topic later.